docs/architecture/architecture.md

```diff
@@ -6,48 +6,71 @@ This document describes the architecture of ibc-rs. If you're looking for a high
 
 ![](docs/architecture/assets/repo-context.png)
```

Comment:
Seems like the picture is not rendered. I would find it useful if we do have an overview diagram, e.g., a system context diagram.